src/shared/lib/navigation/types.ts

```diff
@@ -1,32 +1,32 @@
+import type { BottomTabScreenProps } from '@react-navigation/bottom-tabs';
 import type { CompositeScreenProps } from '@react-navigation/native';
 import type { NativeStackScreenProps } from '@react-navigation/native-stack';
-import type { BottomTabScreenProps } from '@react-navigation/bottom-tabs';
 
 export type RootStackParamList = {
   Login: undefined;
   SignUp: undefined;
   AboutApp: undefined;
+  ChangeLanguage: undefined;
```

Comment:
Well, it was supposed to be the AppLanguage screen. But if you think it should be called ChangeLanguage - I have no problems with that, just please remove AppLanguage